core/src/main/java/org/elasticsearch/action/admin/indices/rollover/RolloverResponse.java

```diff
@@ -39,25 +39,31 @@
     private static final String DRY_RUN = "dry_run";
     private static final String ROLLED_OVER = "rolled_over";
     private static final String CONDITIONS = "conditions";
+    private static final String CREATE_INDEX_ACKNOWLEDGED = "create_index_acknowledged";
```

Comment:
hmm.. i'm not sure we want to expose the complexity of the internal two steps. For example if the alias is acked and the index isn't, it doesn't really mean that the index isn't known by the nodes - having processed the alias change guarantees it.  I think we can keep a simple acknowledge like in other responses, which means all nodes know about the new index and the new alias? it maybe then that with rollover the shards are acked but not the request. If we want to avoid that , I will only wait for shards active after the alias change is done.